### ISSUE TITLE
RSOP does not contain consolidated LcmConfig data

### ISSUE THREAD

**Issue by ykuijs** [dsccommunity/DscWorkshop#94]
When running the Build script, the RSOP output only contains the LcmConfig data that is included in the YAML file of the node itself. The data from the DscBaseline.yml file is missing:
https://github.com/dsccommunity/DscWorkshop/blob/935d168cad1546c4bf37db23f0735380d2d9866d/DSC/DscConfigData/Roles/DscBaseline.yml#L38-L54

**Example of DSCFile01.yml**
```
  MaxLcmRuntime: 00:30:00
LcmConfig:
  ConfigurationRepositoryWeb:
    Server:
      ConfigurationNames: DSCFile01
PSDscAllowDomainUser: true
```

Would be great if these config items would also be included in the RSOP data.

**Comment by raandree** [comment]
Right, this has bothered me quite a few times as well. I am looking into it...

**Comment by raandree** [comment]
Closing this issue and tracking the fix in gaelcolas/datum#95.

Thanks for reporting this, @ykuijs.
